src/evaluators/__tests__/source-4-3.test1.js

```diff
@@ -0,0 +1,2 @@
+parse_and_run('function a(){const b = 2; return b;} a();');
+// 2
```

Comment:
The last line of any text file should be terminated with a newline, by convention. So github gives you this warning:

<img width="462" alt="Screenshot 2020-04-23 at 10 08 48 AM" src="https://user-images.githubusercontent.com/30220768/80051379-a0cbb400-854a-11ea-85d8-deaa1c6bbde2.png">

The newline should not make any difference in the testing, hopefully.